fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseChannel.java

```diff
@@ -58,16 +35,15 @@ public HttpResponseChannel(
      * @param apiRequest  Api request object with all the associated info with it
      * @param httpResponseMaker  Helper class instance to prepare the response object
      *
-     * @deprecated  The ResponseFormatType and UriInfo should be passed explicitly, rather than implicitly via the
```

Comment:
@michael-mclawhorn @archolewa Is really necessary to pass the giant ApiRequest object? It de-separate layer abstraction and makes it more difficult to set up a test. Does passing more variables from ApiRequest solves the problem as well?